modules/tensor_mechanics/test/tests/smeared_cracking/tests

```diff
@@ -8,10 +8,34 @@
     use_old_floor = true
     compiler = 'GCC CLANG'
   [../]
-  [./single_edge_notch]
+  [./cracking]
     type = 'Exodiff'
-    input = 'crack2d.i'
-    exodiff = 'crack2d_out.e'
-    rel_err = 1.0e-03
+    input = 'cracking.i'
+    exodiff = 'cracking_out.e'
+    rel_err = 1e-04
+  [../]
+  [./plane_stress]
+    type = 'Exodiff'
+    input = 'cracking_plane_stress.i'
+    exodiff = 'cracking_plane_stress_out.e'
+    rel_err = 1e-04
+  [../]
+  [./cracking_rz]
+    type = 'Exodiff'
+    input = 'cracking_rz.i'
+    exodiff = 'cracking_rz_out.e'
+    rel_err = 1e-04
+  [../]
+  [./rz_exponential]
+    type = 'Exodiff'
+    input = 'cracking_rz_exponential.i'
+    exodiff = 'cracking_rz_exponential_out.e'
+    rel_err = 1e-04
```

Comment:
This one fails, but it would have passed if you used abs_zero instead of rel_err.